codegen/cmd/injection-gen/generators/informer.go

```diff
@@ -78,6 +80,8 @@ func (g *injectionGenerator) GenerateType(c *generator.Context, t *types.Type, w
 
 	klog.V(5).Info("processing type ", t)
 
+	disableInformerInit, _ := pflag.CommandLine.GetBool("disable-informer-init")
```

Comment:
we should be pulling this from the custom args struct and the funneling it into this generator

https://github.com/knative/pkg/blob/b8b7ca1f790479f3a6ea48084824357b9608d87e/codegen/cmd/injection-gen/generators/packages.go#L41

https://github.com/knative/pkg/blob/b8b7ca1f790479f3a6ea48084824357b9608d87e/codegen/cmd/injection-gen/generators/packages.go#L97

otherwise it's like we're reaching for globals.

The logic in packages is a bit verbose :/

Reply:
It is ok I will adjust it.

Reply:
@dprotaso done.